src/app/shared/utils/parse-time-holes.ts

```diff
@@ -9,16 +9,18 @@ export function parseTimeHoles(
   timeWindowEnum: string,
   timeHoles = true
 ) {
-  if (timeHoles === false) {
-    return dataset.reduce((acc, d: SingleSample | MultiSample, i, arr) => {
-      const date = new Date(d.startDateTime)
+  // NOTE: If timeholes is false only add timeholes at edges
 
-      // --> Add the current dataset value
-      acc.push({ date, value: d.value })
+  // if (timeHoles === false) {
+  //   return dataset.reduce((acc, d: SingleSample | MultiSample, i, arr) => {
```

Comment:
@petcupaula Could you please tell what `timeHoles` means?

Reply:
It's to fill in the gap between data points. Since the API only returns what it has (effectiveTimeFrame) and not our required time frame. And in order to draw the graph, d3 requires strictly defined data.